src/Dotnet.Installer.Console/Scripts/dotnet-launcher.sh

```diff
@@ -2,67 +2,19 @@
 
 set -euo pipefail
 
-# Run a command in elevated mode (sudo or pkexec, whichever is available)
-run_elevated() {
-    # shellcheck disable=SC2153
-    local dotnet_install_dir="$DOTNET_INSTALL_DIR"
-    local should_debug="$1"  # Get the debug value as the first argument
-    shift  # Remove the first argument to keep only the command
+# Check for installed .NET components
+manifest_path="$SNAP_COMMON/snap/manifest.json"
 
-    local command_to_execute=("$@")
+if [[ $("$SNAP"/usr/bin/jq 'length' "$manifest_path") -eq 0 ]]; then
+    echo "Looks like you don't yet have a .NET SDK or Runtime installed."
+    echo "I am downloading and installing the latest LTS SDK for you to use. It should only be a few moments."
 
-    # Check if the script is running as root
-    if [[ "$EUID" -eq 0 ]]; then
-        "${command_to_execute[@]}"
-    else
-        sudo --preserve-env=DOTNET_INSTALL_DIR,DOTNET_INSTALLER_DEBUG "${command_to_execute[@]}"
-    fi
-
-    return $?
-}
-
-# Main script execution
-if [[ ! -e "/snap/dotnet-manifest/current/supported.json" ]]; then
-    echo "Welcome to .NET on Snap!"
-    echo "Looks like this is your first time running snapped .NET, hold tight while I check which .NET versions are available for you."
-    if ! run_elevated "0" snap install dotnet-manifest; then
-        echo "We could not install the manifest content snap, please check your credentials or run this command with sudo."
+    if ! "$SNAP/Scripts/dotnet-installer-launcher.sh" install sdk lts; then
+        echo "Could not install the latest .NET SDK. Please check your credentials or run this command with sudo."
```

Comment:
issue: The error text states that it failed to install the latest SDK although it failed to install the latest **LTS** SDK:

```suggestion
        echo "Could not install the latest .NET LTS SDK. Please check your credentials or run this command with sudo."
```

Reply:
fixed with: [b9bf857](https://github.com/canonical/dotnet-snap/pull/24/commits/b9bf8578a86c8a75f1bcbc1236dfb566c3001519)